src/v1/steward.ts

```diff
@@ -664,7 +635,7 @@ export function handleCollectPatronage(event: CollectPatronage): void {
     steward,
     tokenPatron,
     txTimestamp,
-    false
+    false // why is this false? surely if the deposit is reduced, the foreclosureTime needs to be updated?
```

Comment:
Can't answer off-hand, maybe there is another event that handles this somewhere else?